ros_packages/motors/motors/motor_control.py

```diff
@@ -75,6 +76,9 @@ def __init__(self):
             MotorSettings, "motor_settings", 10
         )
 
+        # Service for Pose
+        self.srv = self.create_service(ApplyPose, "apply_pose", self.apply_pose)
```

Comment:
I don't think we need an additional apply-pose-service and instead should use the existing `apply_joint_trajectory`-service